Crash report

### Expected behaviour
- MD file should open after downloading to internal storage and manually opening it using the menu in app

### Actual behaviour
- Crash and optional prompt to send report

### Environment data

### Logs

#### Nextcloud log (data/nextcloud.log)
```
************ CAUSE OF ERROR ************

java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$3.done(AsyncTask.java:318)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:243)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
	at java.lang.Thread.run(Thread.java:761)
Caused by: java.lang.NoSuchMethodError: No virtual method toPath()Ljava/nio/file/Path; in class Ljava/io/File; or its super classes (declaration of 'java.io.File' appears in /system/framework/core-oj.jar)
	at org.mozilla.universalchardet.UniversalDetector.detectCharset(UniversalDetector.java:352)
	at org.mozilla.universalchardet.ReaderFactory.createBufferedReader(ReaderFactory.java:60)
	at org.mozilla.universalchardet.ReaderFactory.createReaderFromFile(ReaderFactory.java:92)
	at org.mozilla.universalchardet.ReaderFactory.createReaderFromFile(ReaderFactory.java:104)
	at com.owncloud.android.ui.preview.PreviewTextFileFragment$TextLoadAsyncTask.doInBackground(PreviewTextFileFragment.java:192)
	at com.owncloud.android.ui.preview.PreviewTextFileFragment$TextLoadAsyncTask.doInBackground(PreviewTextFileFragment.java:162)
	at android.os.AsyncTask$2.call(AsyncTask.java:304)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	... 4 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30190090
Build flavor: generic

************ DEVICE INFORMATION ************
Brand: lge
Device: lv7
Model: LG-M327
Id: NRD90U
Product: lv7_crk_us

************ FIRMWARE ************
SDK: 24
Release: 7.0
Incremental: 190581602ce2d

```

--- a/build.gradle
+++ b/build.gradle
@@ -263,7 +263,7 @@ dependencies {
     implementation "androidx.work:work-runtime:$workRuntime"
     implementation "androidx.work:work-runtime-ktx:$workRuntime"
     implementation "androidx.fragment:fragment-ktx:1.4.1"
-    implementation 'com.github.albfernandez:juniversalchardet:2.3.2' // need this version for Android <7
+    implementation 'com.github.albfernandez:juniversalchardet:2.0.3' // need this version for Android <7
     compileOnly 'com.google.code.findbugs:annotations:3.0.1u2'
     implementation 'commons-io:commons-io:2.11.0'
     implementation 'org.greenrobot:eventbus:3.3.1'

